api/src/test/java/org/openmrs/api/ProviderServiceTest.java

```diff
@@ -317,7 +317,7 @@ public void saveProviderAttributeType_shouldSaveTheProviderAttributeType() throw
 	 */
 	@Test
 	public void unretireProvider_shouldUnretireAProvider() throws Exception {
-		Provider provider = service.getProvider(1);
+		Provider provider = service.getProvider(2);
```

Comment:
Do you mind explaining a bit why this changed necessitated a switch from 1 to 2 for provider id?